src/app/features/stacks-transaction-request/stacks-transaction-signer.tsx

```diff
@@ -111,6 +103,15 @@ export function StacksTransactionSigner({
       >
         {() => (
           <>
+            <PageTop />
```

Comment:
Maybe we can extract these components into another file so we can include that in `stacks-transaction-signer.tsx` and `transaction-request.tsx` instead of repeating these lines

Reply:
Yeah, we need to refactor our transaction flows for sure, agree lots of duplicate code. I created a code health issue for it here: https://github.com/leather-wallet/extension/issues/3594